Makefile

```diff
@@ -68,6 +68,29 @@ checkgenerate:
 	@# Used in CI to verify that `make generate` doesn't produce a diff.
 	test -z "$$(git status --porcelain | tee /dev/stderr)"
 
+.PHONY: runservertests
+runservertests: $(BIN)/connectconformance $(BIN)/referenceserver
+	$(BIN)/connectconformance --mode server $(BIN)/referenceserver
+
+.PHONY: runclienttests
+runclienttests: $(BIN)/connectconformance $(BIN)/referenceclient
+	$(BIN)/connectconformance --mode client $(BIN)/referenceclient
```

Comment:
We probably want to add these to the existing `test` target, so running tests in CI will also run both of these.

Reply:
I added these to the `all` target so that `make` runs them. I also added them as a separate call in `ci.yaml` so that we could see them as a separate step in the Github Action. It felt strange to mix them with the `make test` target. Even though they're tests, it felt strange to mix them with the `make test` target since they're a totally different kind of tests. Not a big deal to me though. If you'd rather I put them in `make test` I can do that too, just lmk.